### PR TITLE
feat: Add contact info

### PR DESCRIPTION
**Pull request by novellac**
## Context

[Jira card #43](https://ywgc.atlassian.net/jira/software/projects/YWGC/boards/1?selectedIssue=YWGC-43)
The client wasn't set up to use a phone number yet, so I've added an email as the contact.

## Work done
I added the contact info for YWGC to the mobile header, the desktop header and the services area.

## Manual testing instructions
1. Serve locally or go to the netlify deploy link below
2. Resize the site so it is in mobile layout.
3. Navigate to the top of the page
- Expect to see the "Tap to email" with the email listed.
4. Resize the site so it is in desktop layout.
5. Navigate to the top of the page
- Expect to see the text "Email Us!" in the header, as a link to the email.
6. Navigate to the services area.
- Expect to see the text, "Send us an email" as a link to the email.

## Gotchas/What I learned
I always forget the syntax to write a email as a link. It is: `<a href="mailto:info@yourcoolemail.com">Email us today!</a>`

### CODE REVIEW

**Review by jaynaxissa — COMMENTED** [review]
looks fine to me! Could we check with the client later if they still want to use their phone number?

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -8,12 +8,12 @@ function Home () {
...
-            Tap to call!
-          </p>
-          <p className='font-normal font-header text-xl text-green-700 mx-2'>
-            123.456.7890
+          <p className='mx-2 ml-auto text-xl font-normal font-header'>
+            Tap to email!
           </p>
+          <a href="mailto:info@yorkshirewoodsgroundcrew.com" className='mx-2 text-xl font-normal text-green-700 font-header'>
+            info@yorkshirewoodsgroundcrew.com
+          </a>
```
Comment (NurkaAmre) [inline src/pages/Home.jsx:16]: It looks like the link in the mobile version might not be positioned quite right; we should make sure it fits well on smaller screens.

mobile

**Review by NurkaAmre — DISMISSED** [review]
### Hello @novellac ✋ 
***
The Mobile Version text does not fit with the screen

**Comment by novellac** [comment]
> looks fine to me! Could we check with the client later if they still want to use their phone number?

They are only using email for now.

File: src/pages/Home.jsx (hunk 1 above)
Comment (novellac) [reply src/pages/Home.jsx:16]: Good catch. I got updated designs from the designer so we can avoid this.

**Review by ajkarczewski — APPROVED** [review]
Looks good!

**Review by ajkarczewski — APPROVED** [review]

**Pull request merged**